static/app/components/events/eventReplay/replayPreview.tsx

```diff
@@ -120,6 +127,15 @@ function ReplayPreview({orgSlug, replaySlug, eventTimestampMs, buttonProps}: Pro
     },
   };
 
+  const fullReplayUrlBreadcrumbs = {
+    pathname: normalizeUrl(`/organizations/${orgSlug}/replays/${replayId}/`),
+    query: {
+      referrer: getRouteStringFromRoutes(routes),
+      t_main: TabKey.BREADCRUMBS,
+      t: initialTimeOffsetMs / 1000,
+    },
+  };
```

Comment:
l: you could avoid allocating an object if you moved the ternary here e.g.

```
const openReplayLink = fromFeedback ? {...} : {...}
```

Or really, it's only `t_main` that changes between the two

Reply:
ahh true thank you